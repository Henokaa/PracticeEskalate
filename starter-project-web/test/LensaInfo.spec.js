import { mount, createLocalVue } from '@vue/test-utils'
import LensaInfo from '@/components/LensaInfo.vue'
import Vuetify from 'vuetify'
const localVue = createLocalVue()

describe('LensaInfo', () => {
  let vuetify
  beforeEach(() => {
    vuetify = new Vuetify()
  })

  test('renders properly and matchs snapshoot', () => {
    const wrapper = mount(LensaInfo, { vuetify, localVue })
    expect(wrapper.text()).toContain('Lensa Billion')
    expect(wrapper.html()).toMatchSnapshot()
  })
})
