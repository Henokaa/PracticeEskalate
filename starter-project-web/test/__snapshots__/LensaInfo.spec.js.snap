// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LensaInfo renders properly and matchs snapshoot 1`] = `
"<v-row dense=\\"\\" no-gutters=\\"\\">
  <v-col cols=\\"12\\" sm=\\"6\\" md=\\"3\\">
    <v-card outlined=\\"\\" class=\\"white black--text pa-2\\">
      <v-avatar color=\\"grey\\" size=\\"250px\\" class=\\"profile\\"><img contain=\\"\\" src=\\"../assets/lensa.jpg\\" class=\\"fill-height bottom-gradient\\"></v-avatar>
    </v-card>
  </v-col>
  <v-col md=\\"8\\">
    <v-card outlined=\\"\\" class=\\"white black--text pa-2\\">
      <v-card-title class=\\"text-h5\\">
        Lensa Billion
      </v-card-title>
      <v-card-text class=\\"black--text\\">
        I am a third year Software Engineering student at AAiT. In my
        leisure I like to read biographical and philosophical novels. I
        also enjoy listening to music and eating. I am a great admirer
        of Ethiopian cultural dances. So much so I try to learn and
        perform these dances to the best of my abilities.
      </v-card-text>
    </v-card>
  </v-col>
</v-row>"
`;
